docs/cow-protocol/reference/core/auctions/ebbo_specifics.md

```diff
@@ -0,0 +1,49 @@
+---
+id: ebbo-rules
+---
+
+# EBBO violations: details
+
+In this section, we will elaborate on the details about what consistutes an EBBO violation, and what actions are taken by the DAO in case such a violation occurs. The content of this section is based on [CIP-52](https://snapshot.org/#/cow.eth/proposal/0x0f2f1fde68d85081a7d60f7ac99dafbdabdbf8c8cf55961f2609b3dff429a24a).
+
+
+## Certificate of EBBO violation
+
+A certificate for an EBBO violation consists of a reference routing on a block (and log index) between the start of the auction and when the settlement happened onchain. A reference routing for a trade at a given block (and index) is an execution of that trade which only uses liquidity from base protocols and routes through base tokens (see [here](/cow-protocol/reference/core/auctions/competition-rules#base-tokens) for the definition of base protocols and base tokens). The surplus received by users in this routing is used as reference surplus. The difference between the reference surplus and the surplus actually received by the user is the size of the EBBO violation. This amount needs to be reimbursed to a user.
+
+A certificate for an EBBO violation can be challenged by the solver who is accused of the EBBO violation by providing a different block (and index), within 72 hours of the notification of the violation. In this case, a reference routing on this block (and index) might be proposed by the core team and used as certificate instead. The new certificate, if any, cannot be challenged again.
+
+## Reimbursement procedures
+
+The following steps are taken, in case an EBBO violation is detected, either by the core team’s monitoring infrastructure or a third party reach out (for example by the user themselves or by solvers). We clarify here that in order to avoid reports of very old trades that occurred at a time where the protocol was in earlier phases, violations will be inspected only if they are communicated within 3 months from the incident date.
+
+1. A violation is detected, within 3 months from the incident date.
```

Comment:
I guess it's too late, but 3 months is a very loooong time. 30 days would have been fine to imo.

Reply:
Yeah this is already set in the CIP so can't change it now.